test/jdk/java/awt/print/PrinterJob/PrintNullString.java

```diff
@@ -21,164 +21,112 @@
  * questions.
  */
 
-import java.awt.Button;
-import java.awt.Dimension;
-import java.awt.Frame;
 import java.awt.Graphics;
 import java.awt.Graphics2D;
-import java.awt.Panel;
 import java.awt.print.PageFormat;
 import java.awt.print.Printable;
 import java.awt.print.PrinterException;
 import java.awt.print.PrinterJob;
+import java.io.File;
 import java.text.AttributedCharacterIterator;
 import java.text.AttributedString;
 
-import javax.swing.JOptionPane;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.print.attribute.standard.Destination;
 
 /*
  * @test
  * @bug 4223328
- * @summary Printer graphics must behave the same as screen graphics
+ * @summary Printer graphics must throw expected exceptions
  * @key printer
- * @library /java/awt/regtesthelpers
- * @build PassFailJFrame
- * @run main/manual PrintNullString
+ * @run main PrintNullString
  */
-public class PrintNullString extends Frame {
-    private static final String INSTRUCTIONS =
-            "This test should print a page which contains the same\n" +
-            "text messages as in the test window on the screen.\n" +
-            "\n" +
-            "The messages should contain only 'OK' and 'expected' messages.\n" +
-            "Press Pass if it's the case; otherwise press Fail.\n" +
-            "\n" +
-            "If the page fails to print, but there were no exceptions\n" +
-            "then the problem is likely elsewhere (i.e. your printer)";
+public class PrintNullString implements Printable {
+    private final String nullStr = null;
+    private final String emptyStr = "";
+    private final AttributedString emptyAttStr = new AttributedString(emptyStr);
+    private final AttributedCharacterIterator nullIterator = null;
+    private final AttributedCharacterIterator emptyIterator = emptyAttStr.getIterator();
 
     public static void main(String[] args) throws Exception {
         if (PrinterJob.lookupPrintServices().length == 0) {
             throw new RuntimeException("Printer not configured or available.");
         }
-
-        PassFailJFrame.builder()
-                .instructions(INSTRUCTIONS)
-                .testUI(PrintNullString::new)
-                .rows((int) INSTRUCTIONS.lines().count() + 1)
-                .columns(45)
-                .build()
-                .awaitAndCheck();
+        new PrintNullString();
```

Comment:
```suggestion

        new PrintNullString();
```
Leave a blank line between the skipped condition and the real test code.